xml/MAIN-SBP-CSP-UpdateInfra.xml

```diff
@@ -7,469 +7,765 @@
-		<!--  <orgname>SUSE Best Practices</orgname>-->
-		<productname>SUSE Linux Enterprise Server 15 SP1, Repository Mirroring Tool</productname>
-		<!-- <productnumber>15 SP1</productnumber>-->
-		<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
-			<dm:bugtracker>
-				<dm:url>https://github.com/SUSE/suse-best-practices/issues/new</dm:url>
-				<dm:product>SUSE Update Infrastructure Setup Guide for Cloud Service
-					Providers</dm:product>
-			</dm:bugtracker>
-			<dm:editurl>https://github.com/SUSE/suse-best-practices/edit/main/xml/</dm:editurl>
-		</dm:docmanager>
-		<author>
-			<personname>
-				<firstname>Jona</firstname>
-				<surname>Apelbaum, CSP Cloud Architect, SUSE</surname>
-			</personname>
-			<!--   <affiliation>-->
-			<!--    <jobtitle>Public Cloud Architect</jobtitle>-->
-			<!--    <orgname>&suse;</orgname>-->
-			<!--   </affiliation>-->
-		</author>
-		<author>
-			<personname>
-				<firstname>Mike</firstname>
-				<surname>Friesenegger, Solution Architect, SUSE</surname>
-			</personname>
-			<!--   <affiliation>-->
-			<!--    <jobtitle>Solution Architect</jobtitle>-->
-			<!--    <orgname>&suse;</orgname>-->
-			<!--   </affiliation>-->
-		</author>
-		<date>February 08, 2021</date>
-		<abstract>
-			<para> This guide describes the setup of the recommended infrastructure for offering
-				SUSE product as on-demand offerings in a cloud environment. The update
-				infrastructure scales from small to large cloud installations. The guide is an
-				optional, but highly recommended element of a Cloud Service Provider's (CSPs)
-				infrastructure. It allows for central license and repository management. </para>
-		</abstract>
-	</info>
-	<sect1 xml:id="sec-intro">
-		<title>Introduction</title>
+ <info>
+  <title>SUSE Update Infrastructure Setup Guide for Cloud Service Providers</title>
+<!--  <orgname>SUSE Best Practices</orgname>--><productname>SUSE Linux Enterprise Server 15 SP3, Repository Mirroring Tool</productname>
+<!-- <productnumber>15 SP3</productnumber>-->
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+    <dm:url>https://github.com/SUSE/suse-best-practices/issues/new</dm:url>
+    <dm:product>SUSE Update Infrastructure Setup Guide for Cloud Service Providers</dm:product>
+   </dm:bugtracker>
+   <dm:editurl>https://github.com/SUSE/suse-best-practices/edit/main/xml/</dm:editurl>
+  </dm:docmanager>
+  <author><personname><firstname>Bryan</firstname><surname>Morriss, MSP Cloud Architect, SUSE</surname></personname>
+<!--   <affiliation>-->
+<!--    <jobtitle>Public Cloud Architect</jobtitle>-->
+<!--    <orgname>&suse;</orgname>-->
+<!--   </affiliation>-->
+  </author>
+  <author><personname><firstname>Mike</firstname><surname>Friesenegger, Solution Architect, SUSE</surname></personname>
+<!--   <affiliation>-->
+<!--    <jobtitle>Solution Architect</jobtitle>-->
+<!--    <orgname>&suse;</orgname>-->
+<!--   </affiliation>-->
+  </author><date>September 03, 2021</date>
```

Comment:
@rjschwei Hi Robert,
it is a bit of a challenge if authors leave SUSE - but are not responsible anymore - to still list their names on the paper - and this is for several reasons: 

- One of the main reasons why we add (SUSE) author names to best practices is to know whom to approach if changes or updates are needed. If an author left - this information is useless at least for this purpose
- If we would keep authors listed who left SUSE, after an update of the paper, would we list them with their previous SUSE job function, or with their new company and job function? The first option would simply be wrong, the second option is not really practicable because we would need to ask for permission from their new company
- Also,  if an author inside SUSE changes roles, and another one takes over subject matter expertise, and from now on is "responsible" for a tech document, should we have the colleague NOT being responsible anymore still listed? If responsibilities change often, the list of authors could become veeeeeery long.
- Same is valid in general for (shorter) technical docs referring to a specific product version / SP / software, as usually their "half-life period" is also much shorter than for example for books, and updates are happening more frequently...

Thus, we decided to list only the *actual* or *currently* responsible authors and SME's. For SUSE papers, as everything written during work time and with SUSE equipment is SUSE property anyway, we would not need to list authors. But we do that for the SBP's because we think it is just fair to give them at least some credits for the creation of a document, and for their (current) work on a technical solution.

Hope this explains it better. 
Thanks very much - Meike

Reply:
Yes, I understand all the reasoning, which is why referring to teams is a great why to avoid having to follow the bouncing ball of personal career choices.